<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryPanel</class>
 <widget class="QWidget" name="CategoryPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="categoryLayout">
   <property name="spacing">
    <number>0</number>
   </property>

   <item>
    <widget class="QScrollArea" name="categoryScrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="categoryContainer">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>782</width>
        <height>69</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>